apps/meteor/server/services/translation/service.ts

```diff
@@ -22,7 +22,7 @@ export class TranslationService extends ServiceClassInternal implements ITransla
 	}
 
 	// Use translate when you want to translate to the user's language, or server's as a fallback
-	async translate(text: string, user: IUser): Promise<string> {
+	async translate(text: string, user: AtLeast<IUser, 'language'>): Promise<string> {
```

Comment:
Let's dismiss this unrelated change, also, according to the function implementation the language is not required.